lua/lspconfig/server_configurations/kotlin_language_server.lua

```diff
@@ -39,28 +42,16 @@ return {
     You could refer for this capability to:
     https://github.com/udalov/kotlin-vim (recommended)
     Note that there is no LICENSE specified yet.
+
+    For faster startup, you can setup caching by specifying a storagePath
+    in the init_options. The default is your home directory.
     ]],
     default_config = {
-      root_dir = [[root_pattern("settings.gradle")]],
+      root_dir = [[See source]],
       cmd = { 'kotlin-language-server' },
-      capabilities = [[
```

Comment:
This section was not rendered in docs.
![image](https://github.com/neovim/nvim-lspconfig/assets/20788334/603d31a8-9573-45a8-8377-41daa436c2ec)